operations/research-and-development/product/release-process/mobile-release.md

```diff
@@ -8,135 +8,12 @@ Please refer to [the Mobile Release Playbook](https://community.mattermost.com/p
-
-### F. \(T-minus 2 working days\) Release Build Cut
-
-The final release is cut. If an urgent and important issue needs to be addressed between major releases, a bug fix release \(e.g., 1.1.1\) may be created.
-
-1. Release Manager:
-   * Post this checklist in Release Checklist channel
-   * Verify all items in the last posted release checklist are complete
-   * Submit Changelog PR for review
-     * Merge Changelog PR after review is complete
-2. Build: 
-   * Tags the release and cuts the final build
-   * Upload the `.ipa` and `.apk` files to GitHub
-3. Marketing:
-   * Finish draft of marketing images and send to marketing lead for review
-4. Docs:
-   * Finalize docs
-     * If reviews are not complete, hold a 30-minute doc review meeting with PMs and anyone else who has changed or reviewed docs this release and wants to join
-     * Submit a correction PR for any incorrect formatting or other errors missed during the initial review
-
-### G. \(T-minus 0 working days\) Release Day and Build Submitted to App Stores
-
-1. Release Manager:
-   * Post this checklist in Release Checklist channel
-   * Verify all items in the last posted release checklist are complete
-   * Submit final build to Google Play Store and iTunes Connect
-   * Confirm that minimum server version required is clear in update notes in the App stores and in GitHub README and Releases pages
-   * Close the release in Jira
-   * Schedule a release retrospective meeting, to be held within five days from the release 
-   * Post key dates for the next release in the header of the Native Mobile Apps channel and remove links to RC candidates and testing spreadsheet
-     * Make sure that statutory holidays for Canada and US are accounted for in the release dates
-   * Check for any [UserVoice](https://docs.google.com/spreadsheets/d/1nljd4cFh-9MXF4DxlUnC8b6bdqijkvi8KHquOmK8M6E/edit#gid=0) feature suggestions that were completed in the current release
-2. Build:
-   * Merge the release branch back in to master
-   * Review and update project dependencies as needed
-   * Create release in GitHub
-
-### H. \(T-plus X days\) Release Marketing
-
-1. Marketing:
-   * After the apps are approved and on the App Store \(number of days may vary\), send out release marketing
-     * Update the app version on the [download page](https://mattermost.com/download/)
-     * Send out a Twitter announcement
-   * Double check that minimum server version required is clear in update notes in the App stores and in GitHub README and Releases pages
-
+Note: T-minus counts are measured in "working days" \(weekdays, Monday through Friday, excluding [the listed statutory holidays](https://handbook.mattermost.com/operations/workplace/people/working-at-mattermost/paid-time-off#holidays)\) prior to release day.
+
+**Schedule for Mobile App releases**:
+ - Feature Complete deadline is approximately 1 month prior to the release day.
+ - Cut release branch based off `main` and cut beta builds (around T-14).
```

Comment:
Once the release branch is created, how often do we want to create builds (release candidates of sort) for QA? We can keep creating betas from main, but as stated before, once the release branch is created, the beta (out of main) may contain code that is not meant to release. So on those 14 days since the release branch is created, at least 1 beta will be released out of sync with the release.

In the meanwhile, some fixes may be cherry picked to the release branch.

We could create a new build after every cherry pick, or a more forgiving cadence (once a day if there has been any merge, or even once a week). 0/5 on the final details, but I feel we should at least discuss it.

Reply:
Adding @lindalumitchell & @saturninoabril for any thoughts.

Reply:
Thanks, I think at most one new build per day for testing (if there have been changes / cherry picks of course) would be often enough.

Reply:
> Once the release branch is created, how often do we want to create builds (release candidates of sort) for QA?

+1 to what Linda said.

@larkox How do we know if the build is from main or release branch during the release testing period or in general? How about alpha for main and beta build for release branch?

Reply:
I will leave this to @enahum , since changing the name of the app to support this, may not be possible (or practical). We could potentially add a new parameter to the build to use on the build info (build name?) along the build number, build version, etc... But not sure that is the best idea neither.

Reply:
Oh boy, this is getting out of hand and will create a lot of confusion, it already started with community members.

Cutting builds have an impact, specially on the stores so having a cadence of daily builds is not something that I will encourage.

Having a 3rd app is also not an option, 2 should be more than enough as it allows us to have the latest and gratest as well as what is actually being shipped.

The idea is to cover our based but without over complicating things.

As I mentioned in the QA channel, the idea is to coordinate between the teams to make minimize the pain, not to add more :) 

That said, we can build as many as we want like PR's that do not increase the build numbers nor are submitted to the stores (including testflight)

Reply:
I've made edits to [the Mobile release process section](https://github.com/mattermost/mattermost-handbook/pull/1192/files#diff-2589aadc650609878898e858a1cd2ce69dd31bba3c001f94be7182d8f797a031) based on the discussions here.